.github/workflows/pr_merge_cherry-markdown_npm-dev.yml

```
@@ -59,7 +59,7 @@ jobs:
       - name: Set up Node.js
         uses: actions/setup-node@v4
         with:
-          node-version: lts/*
+          node-version: 18
```

挪到.node-version，放在根目录
```suggestion
          node-version-file: .node-version
```

---

这个目前不添加，还是以 `package.json` 的 `engines` 为限制，不影响开发场景下的 node 版本选择